server/mailet/mailets/src/main/java/org/apache/james/transport/matchers/dlp/DlpRulesLoader.java

```diff
@@ -40,7 +42,7 @@ public Impl(DLPConfigurationStore configurationStore) {
 
         @Override
         public DlpDomainRules load(Domain domain) {
-          return toRules(configurationStore.list(domain));
+          return toRules(Mono.from(configurationStore.list(domain)).block());
```

Comment:
can we make this method reactive and avoid blocking ?

Reply:
In another PR